buildenv/jenkins/JenkinsfileBase

```diff
@@ -56,6 +56,8 @@ def setupEnv() {
 			env.JCK_VERSION = "jck9"
 		} else if ( JAVA_VERSION == "SE100" ) {
 			env.JCK_VERSION = "jck10"
+		} else if ( JAVA_VERSION == "SE110" ) {
+			env.JCK_VERSION = "jck11"
 		}
```

Comment:
I think we can do a regex, as for most cases, its a straight replacement (jck8b being the one exception).  See what I have done here: https://github.com/smlambert/openjdk-tests/blob/jckUpdatesV2/buildenv/jenkins/JenkinsfileBase#L55  to handle this (and hopefully not have to keep updating each new version)

Reply:
Agreed, will look into it.

Reply:
Fixed and verified manually at local docker image.
Still attempting a personal build to verify this PR in a whole.